src/Protocol/TextEdit.php

```diff
@@ -22,4 +22,10 @@ class TextEdit
      * @var string
      */
     public $newText;
+
+    public function __construct(Range $range = null, string $newText)
```

Comment:
Non-optional parameter cannot follow an optional param. Both should be optional to allow constructing a instance and then setting the properties manually